src/repositories/scrapbookRepository/index.ts

```diff
@@ -1,47 +0,0 @@
-import { prisma } from '../../config';
-
-async function create(name: string, userId: number, numberPictures: number) {
-  return prisma.scrapbook.create({
-    data: { name, userId, numberPictures },
-  });
-}
-
-async function findScrapbookById(id: number) {
-  return prisma.scrapbook.findUnique({
-     where: {
-      id,
-    },
-   
-    include:{      
-      Pictures:{orderBy:{createdAt:"asc"}}       
-    },
-    
-  });
-}
-
-async function findScrapbookByUserId(userId: number) {
-  return prisma.scrapbook.findMany({
-    where: {
-      userId,
-    },include:{
-      Pictures:{orderBy:{createdAt:"asc"}} 
-    }
-  });
-}
-
-async function deleteScrapbook(id: number) {
-    return prisma.scrapbook.delete({
-        where:{
-            id,
-        }
-    })
-}
```

Comment:
Nesses itens, as funções não precisam ter scrapbook no nome, pois o repositorio em si já é chamado de scrapbookRepository, ou seja, o que for procurar vai ser em scrapbook e o que for deletar também vai. só coloque nome diferente caso ele utilize alguma outra tabela em conjunto por questão de performance. ex:
findByIdAndPictureCategory() - (o que deve ser evitado a não ser que seja aja um estudo forte relacionado a performance em cima)